File: android/src/main/java/com/dexterous/flutterlocalnotifications/models/styles/MediaStyleInformation.java
package com.dexterous.flutterlocalnotifications.models.styles;

import java.util.ArrayList;

public class MediaStyleInformation extends DefaultStyleInformation {
    public ArrayList<Integer> showActionsInCompactView;

    public MediaStyleInformation(ArrayList<Integer> showActionsInCompactView, Boolean htmlFormatTitle, Boolean htmlFormatBody) {
        super(htmlFormatTitle, htmlFormatBody);
        this.showActionsInCompactView = showActionsInCompactView;
    }
}

File: lib/src/platform_specifics/android/styles/media_style_information.dart
import 'default_style_information.dart';

class MediaStyleInformation extends DefaultStyleInformation {

  MediaStyleInformation({
    bool htmlFormatContent = false,
    bool htmlFormatTitle = false,
  }) : super(htmlFormatContent, htmlFormatTitle);

  Map<String, dynamic> toMap() {
    var styleJson = super.toMap();
    return styleJson;
  }
}
